app/src/main/java/com/duycuong/weather/utils/Constant.java

```diff
@@ -21,6 +21,9 @@
     public static final String HTTP_CODE = "HTTP Code";
     public static final String UTF_8 = "UTF-8";
 
+    public static final String ddMM = "dd/MM";
```

Comment:
ERROR: Static final field names must either be all caps (e.g. int HEIGHT_PX) for true constants, or start with s (e.g. AtomicInteger sNextId or Runnable sSuspendTask) for fields with mutable state or that dont feel like constants.